embeddings/transformation/hf_transformation/sampling_transformation.py

```diff
@@ -81,3 +83,43 @@ def transform(self, data: datasets.DatasetDict) -> datasets.DatasetDict:
             return data
 
         return dataset
+
+
+class SampleSplitsHuggingFaceTransformation(SampleSplitTransformation):
+    def _train_test_split(
+        self, data: datasets.Dataset, test_fraction: float
+    ) -> datasets.DatasetDict:
+        return data.train_test_split(test_size=test_fraction, seed=self.seed)  # type: ignore
+
+
+class SampleSplitsSklearnTransformation(SampleSplitTransformation):
```

Comment:
maybe it is worth to consider to add "stratified" keyword to this transformation. And adjust configuration to stratification as default?

Reply:
I was also wondering whether we need two different transformations for the sample split? We could simply replace the huggingface one with the sklearn one

Reply:
In current situation yes, as we used HF SampleSplitTransformation in experiments. Therefore I would go with StratifiedSamplingTransformation with sklearn